src/main/resources/model/instance.json

```diff
@@ -26,23 +26,24 @@
       "type": "object",
       "properties": {
         "alternativeTitle": {
-          "inventorySearchTypes": "title",
+          "inventorySearchTypes": ["title", "keyword"],
           "index": "multilang"
         }
       }
     },
     "indexTitle": {
-      "inventorySearchTypes": "title",
+      "inventorySearchTypes": ["title", "keyword"],
       "index": "multilang"
     },
     "series": {
-      "inventorySearchTypes": "title",
+      "inventorySearchTypes": ["title", "keyword"],
       "index": "multilang"
     },
     "identifiers": {
       "type": "object",
       "properties": {
         "value": {
+          "inventorySearchTypes": ["title", "keyword"],
```

Comment:
identifiers should be only keyword, without title search group

Reply:
thanks, removed.